File: 2553. Separate the Digits in an Array.rb
#* 2553. Separate the Digits in an Array
nums = [13,25,83,77]
# @param {Integer[]} nums
# @return {Integer[]}
def separate_digits(nums)
  p nums.join.split('').map(&:to_i)
    
end
p separate_digits(nums)

File: 2553. Separate the Digits in an Array.js
// 2553. Separate the Digits in an Array
nums = [13,25,83,77]
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var separateDigits = function(nums) {
  return nums.join('').split('')
};
console.log(separateDigits(nums))